test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -858,6 +862,86 @@ class V2DeparturesControllerSpec
 
   }
 
+  // for retrieving a single departure/movement
+  "GET  /movements/departures/:departureId" - {
```

Comment:
Is it worth putting a Tech Debt ticket into the backlog to rework these tests using model generators? (Or at least to use fewer magic values)

Reply:
Might be easier to just rename the test, so rather than using the URI, I can use the comment above it as the name of the test?